tools/flaky-test-reporter/ghutil/ghutil.go

```diff
@@ -0,0 +1,327 @@
+			var resp *github.Response
+			var err error
+			res, resp, err = client.Issues.GetComment(ctx, org, repo, commentID)
+			return resp, err
+		},
+	)
+	return res, err
+}
+
+// CreateComment adds comment to issue
+func CreateComment(org, repo string, number int, commentStr string) error {
+	comment := &github.IssueComment{
+		Body: &commentStr,
+	}
+	_, err := retry(
+		fmt.Sprintf("commenting issue '%s %s %d'", org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.CreateComment(ctx, org, repo, number, comment)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// EditComment edits comment with given comment ID
+func EditComment(org, repo string, commentID int64, commentStr string) error {
+	comment := &github.IssueComment{
+		Body: &commentStr,
+	}
+	_, err := retry(
+		fmt.Sprintf("editing comment '%s %s %d'", org, repo, commentID),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.EditComment(ctx, org, repo, commentID, comment)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// AddLabelsToIssue adds label on issue
+func AddLabelsToIssue(org, repo string, number int, labelStrs []string) error {
+	_, err := retry(
+		fmt.Sprintf("add labels '%v' to '%s %s %d'", labelStrs, org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.AddLabelsToIssue(ctx, org, repo, number, labelStrs)
+			return resp, err
+		},
+	)
+	return err
+}
+
+// RemoveLabelForIssue in case test failed, remove label auto:lessflaky if exists
+func RemoveLabelForIssue(org, repo string, number int, labelStr string) error {
+	_, err := retry(
+		fmt.Sprintf("remove label '%s' from '%s %s %d'", labelStr, org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			resp, err = client.Issues.RemoveLabelForIssue(ctx, org, repo, number, labelStr)
+			return resp, err
+		},
+	)
+	return err
+}
+
+func updateIssueState(org, repo string, state IssueStateEnum, number int) error {
+	stateString := string(state)
+	issueRequest := &github.IssueRequest{
+		State: &stateString,
+	}
+	_, err := retry(
+		fmt.Sprintf("applying '%s' action on issue '%s %s %d'", stateString, org, repo, number),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			_, resp, err = client.Issues.Edit(ctx, org, repo, number, issueRequest)
+			return resp, err
+		},
+	)
+	return err
+}
+
+func waitForRateReset(r *github.Rate) {
+	if r.Remaining <= tokenReserve {
+		sleepDuration := time.Until(r.Reset.Time) + (time.Second * 10)
+		if sleepDuration > 0 {
+			log.Printf("--Rate Limiting-- Tokens reached minimum reserve %d. Sleeping until reset in %v.\n", tokenReserve, sleepDuration)
```

Comment:
```suggestion
			log.Printf("--Rate Limiting-- GitHub tokens reached minimum reserve %d. Sleeping %ds until reset.\n", tokenReserve, sleepDuration)
```

Reply:
I think you should at least accept the suggestion about the time unit to make the log message clearer.

Reply:
I did accepted it, was messed up by bad merge